aikau/src/main/resources/alfresco/lists/views/layouts/Grid.js

```diff
@@ -257,19 +257,22 @@ define(["dojo/_base/declare",
       resizeCell: function alfresco_lists_views_layouts_Grid__resizeCell(containerNodeMarginBox, widthToSet, node, /*jshint unused:false*/ index) {
          domStyle.set(node, {"width": widthToSet});
          var dimensions = {
-            w: widthToSet,
-            h: null
-         };
-         array.forEach(node.children, lang.hitch(this, "resizeWidget", dimensions));
+               w: widthToSet,
+               h: null
+            },
+            widgetsToResize = query("[widgetId]", node); // Resize all contained widgets, not just immediate children.
```

Comment:
I discussed this with Martin this morning and we felt that the additional code required to resize only the first child with a resize method, along with the need for every widget that contains other widgets to pass on a resize method it receives outweighed the risks of some widgets receiving two resize events.